plugins/openshift/skills/generating-ovn-topology/scripts/check_permissions.py

```
@@ -0,0 +1,231 @@
+    DANGEROUS_PERMS = [
+        ("delete", "pods", "Delete pods"),
+        ("create", "pods", "Create pods"),
+        ("patch", "pods", "Modify pods"),
+        ("update", "pods", "Update pods"),
+        ("deletecollection", "pods", "Bulk delete pods"),
+        ("delete", "deployments", "Delete deployments"),
+        ("create", "deployments", "Create deployments"),
+        ("patch", "deployments", "Modify deployments"),
+        ("delete", "services", "Delete services"),
+        ("create", "services", "Create services"),
+    ]
```

Ruff (0.14.4)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)